client/app/components/calendar/filters.scss

```diff
@@ -0,0 +1,13 @@
+.filter-container {
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.filter-box {
+  display: inline;
+  font-size: 10px;
+  text-align: center;
+  margin: 0 3px;
+  padding: 7px;
+  border-radius: 4px;
+}
```

Comment:
All this styling would most likely be unnecessary if you used `react-bootstrap`'s `<Checkbox />`